tests/rptest/services/redpanda.py

```diff
@@ -4113,7 +4113,7 @@ def is_allowed_log_line(line: str) -> bool:
 
             crash_log = None
             for line in node.account.ssh_capture(
-                f"grep -e SEGV -e Segmentation\\ fault -e [Aa]ssert -e Sanitizer -e 'Aborting on shard' {RedpandaService.STDOUT_STDERR_CAPTURE} || true",
+                f"grep -e SEGV -e Segmentation\\ fault -e [Aa]ssert -e Sanitizer -e 'Aborting on shard' -e 'Recorded crash reason' {RedpandaService.STDOUT_STDERR_CAPTURE} || true",
```

Comment:
Maybe it should be `crash reason to crash file` to catch both the successful and failed cases? We still want to know that it crashed in the failed case.